docs/databases/uploads.md

```diff
@@ -45,7 +45,7 @@ When people upload a CSV to a collection, Metabase will:
 
 ## Primary key auto-generation
 
-Metabase will create an a unique primary key column, called `_mb_row_id`, as the first (left-most) column of the uploaded CSV table. This `_mb_row_id` column will contain automatically generated integers. If the uploaded table already includes a column with a primary key, Metabase will treat that column like any other column. Metabase will also ignore any columns in the upload that have a name that will be in the database with the same name as the auto-generated primary key column (e.g., `_MB row-ID` in the CSV will be `_mb_row_id` or `_MB_ROW_ID` in the database).
+When you upload a CSV, Metabase will create an a unique primary key column, called `_mb_row_id`, as the first (left-most) column of the uploaded CSV table. This `_mb_row_id` column will contain automatically generated integers. Even if the uploaded CSV already includes a column with a primary key, Metabase will treat that column like any other column (that is, Metabase will ignore the fact that the column contains a primary key and instead use the `_mb_row_id` column it created as the table's primary key). Metabase will also ignore any columns in the upload that have a name that will be in the database with the same name as the auto-generated primary key column (e.g., `_MB row-ID` in the CSV will be `_mb_row_id` or `_MB_ROW_ID` in the database).
```

Comment:
```suggestion
When you upload a CSV, Metabase will create an a unique primary key column, called `_mb_row_id`, as the first (left-most) column of the uploaded CSV table. This `_mb_row_id` column will contain automatically generated integers. Metabase will also ignore any columns in the upload that have a name that will be in the database with the same name as the auto-generated primary key column (e.g., `_MB row-ID` in the CSV will be `_mb_row_id` or `_MB_ROW_ID` in the database).
```

I'd suggest removing this sentence because CSVs themselves can't have primary keys. They're just text. Maybe they came from a database where one of the columns were a primary key, but I think users are smart enough to know that we can't detect that because the CSV file doesn't contain any of that information.